SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -1694,6 +1696,44 @@ USpatialActorChannel* ActorSystem::TryRestoreActorChannelForStablyNamedActor(AAc
 	return Channel;
 }
 
+void ActorSystem::InvokeRepNotifies()
+{
+	for (FObjectRepNotifies ObjectRepNotifies : RepNotifiesToSend)
+	{
+		UObject* Object = ObjectRepNotifies.Object.Get();
+		if (!IsValid(Object))
+		{
+			continue;
+		}
+		Worker_EntityId EntityId = NetDriver->PackageMap->GetEntityIdFromObject(Object);
+		if (EntityId == SpatialConstants::INVALID_ENTITY_ID)
+		{
+			continue;
+		}
+		USpatialActorChannel* Channel = NetDriver->GetActorChannelByEntityId(EntityId);
```

Comment:
could be const?

Reply:
can't be done because of `Channel->PostReceiveSpatialUpdate` unfortunately